release-notes.md

```diff
@@ -57,6 +59,70 @@
+| Move 3PC Message Request logic into a separate service | | [INDY-2220](https://jira.hyperledger.org/browse/INDY-2220) |
+| Bump pyzmq to the latest version | | [INDY-2213](https://jira.hyperledger.org/browse/INDY-2213) |
+| Integration of Services: Cleanup | | [INDY-2208](https://jira.hyperledger.org/browse/INDY-2208) |
+| Integrate Checkpointer Service into Replica | | [INDY-2179](https://jira.hyperledger.org/browse/INDY-2179) |
+| Use audit ledger in Checkpoints | | [INDY-2177](https://jira.hyperledger.org/browse/INDY-2177) |
+| Integrate OrderingService into Replica | | [INDY-2169](https://jira.hyperledger.org/browse/INDY-2169) |
+| Integrate PrimarySelector into View Change Service | | [INDY-2167](https://jira.hyperledger.org/browse/INDY-2167) |
+| Integrate view change property-based tests into CI | | [INDY-2150](https://jira.hyperledger.org/browse/INDY-2150) |
+| Integrate and run PBFT View Changer simulation tests with a real implementation | | [INDY-2149](https://jira.hyperledger.org/browse/INDY-2149) |
+| Implement PBFT viewchanger service with most basic functionality | | [INDY-2147](https://jira.hyperledger.org/browse/INDY-2147) |
+| Extract and integrate ConsensusDataProvider from Replica | | [INDY-2139](https://jira.hyperledger.org/browse/INDY-2139) |
+| Extract Checkpointer service from Replica | | [INDY-2137](https://jira.hyperledger.org/browse/INDY-2137) |
+| Extract Orderer service from Replica | | [INDY-2136](https://jira.hyperledger.org/browse/INDY-2136) |
+| Simulation tests for View Changer (no integration) | | [INDY-2135](https://jira.hyperledger.org/browse/INDY-2135) |
+| Implementation: Make PBFT view change working | | [INDY-1340](https://jira.hyperledger.org/browse/INDY-1340) |
+| Implement network, executor, orderer and checkpointer as adaptors for existing codebase | | [INDY-1339](https://jira.hyperledger.org/browse/INDY-1339) |
+| Define Interfaces needed for View Change Service | | [INDY-1338](https://jira.hyperledger.org/browse/INDY-1338) |
+| Modify WriteReqManager to meet Executor interface needs | | [INDY-1337](https://jira.hyperledger.org/browse/INDY-1337) |
+| Stop resetting ppSeqNo (and relying on this) in new view | | [INDY-1336](https://jira.hyperledger.org/browse/INDY-1336) |
+| Enable full ordering of batches from last view that have been already ordered, make execution on replicas that executed them no-op | | [INDY-1335](https://jira.hyperledger.org/browse/INDY-1335) |
+| Balance checker script | | [ST-620](https://sovrin.atlassian.net/browse/ST-620) |
+
+### Known Issues
+| Description | Additional Information | Ticket Number |
+| --- | --- | --- |
+| One node doesn't catch up after promotion | | [INDY-2222](https://jira.hyperledger.org/browse/INDY-2222) |
+| State proofs and audit proofs are not working for token ledger after transaction with FEEs is written | | [ST-623](https://sovrin.atlassian.net/browse/ST-623) |
```

Comment:
`ST-623` and `ST-621` are not know issues, they are fixed

Reply:
Fixed.